### PR TITLE
Functional Tests : Audit - BO - Payment

### PR DESCRIPTION
**Pull request by florine2623**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Functional Tests : Audit - BO - Payment
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | :dolphin: : https://github.com/florine2623/testing_pr/actions/runs/12240534989<br/>:seal: : https://github.com/florine2623/testing_pr/actions/runs/12240549507
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/campaigns/audit/BO/10_payment.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  boLoginPage,
+  boPaymentMethodsPage,
+  type BrowserContext,
+  type Page,
+  utilsPlaywright,
+} from '@prestashop-core/ui-testing';
+
+const baseContext: string = 'audit_BO_payment';
+
+describe('BO - Payment methods', async () => {
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/10_payment.ts]:
```suggestion
describe('BO - Payment', async () => {
```

File: tests/UI/campaigns/audit/BO/10_payment.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        await boPaymentMethodsPage.closeSfToolBar(page);
+    
+        const pageTitle = await boPaymentMethodsPage.getPageTitle(page);
+        expect(pageTitle).to.contains(boPaymentMethodsPage.pageTitle);
+    
+        const jsErrors = utilsPlaywright.getJsErrors();
+        expect(jsErrors.length).to.equals(0);
+      });
+
+      it('should go to \'Preferences\' page', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/10_payment.ts]:
```suggestion
      it('should go to \'Payment > Preferences\' page', async function () {
```

File: tests/UI/campaigns/audit/BO/10_payment.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        await testContext.addContextItem(this, 'testIdentifier', 'goToPaymentMethodsPage', baseContext);
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/10_payment.ts]:
```suggestion
        await testContext.addContextItem(this, 'testIdentifier', 'goToPaymentPreferencesPage', baseContext);
```

File: tests/UI/campaigns/audit/BO/10_payment.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        await preferencesPage.closeSfToolBar(page);
+    
+        const pageTitle = await preferencesPage.getPageTitle(page);
+        expect(pageTitle).to.contains(preferencesPage.pageTitle);
+    
+        const jsErrors = utilsPlaywright.getJsErrors();
+        expect(jsErrors.length).to.equals(0);
+      });
+
+});
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/10_payment.ts]:
```suggestion
});

```

File: tests/UI/campaigns/audit/BO/10_payment.ts [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import {expect} from 'chai';
+import testContext from '@utils/testContext';
+
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/10_payment.ts]:
```suggestion
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Some feedbacks

**Review by Progi1984 — DISMISSED** [review]

**Comment by nesrineabdmouleh** [comment]
Please fix eslint errors

**Review by Progi1984 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Pull request merged**